server/src/coqtop/CoqTop8.ts

```diff
@@ -210,15 +211,26 @@ export class CoqTop extends IdeSlave8 implements coqtop.CoqTop {
     return path.join(this.settings.binPath.trim(), this.settings.coqidetopExe);
   }
 
+  private get scriptPath() {
+    // TODO: rename scriptFile to scriptUri everywhere.
+    let uri = Uri.parse(this.scriptFile)
+    if (uri.scheme == "file")
+      return uri.fsPath;
+    else
+      return undefined
```

Comment:
This is new since last revision and enables stepping through unsaved files (but only outside dune coq top).